notification/src/handler/notification/notification.handler.js

```diff
@@ -270,6 +274,15 @@ function getChangeTransactionStateUpdateAction(
   }
 }
 
+function getChangeTransactionTimestampUpdateAction(transactionId) {
+  const currentTimestamp = new Date()
+  return {
+    action: 'changeTransactionTimestamp',
+    transactionId,
+    timestamp: currentTimestamp.toISOString(),
```

Comment:
In the notification we have a field `eventDate` with timestamp. Why did you choose to create the timestamp in code and not taking it from the notification? https://docs.adyen.com/api-explorer/#/Webhooks/v1/post/REFUND

Reply:
I thought we take actual transaction create/update moment as timestamp
https://github.com/commercetools/commercetools-adyen-integration/pull/826/commits/63c5e10a57b035d00ab5596aaf966e45d53a3130

Reply:
I'm just asking why, maybe you're right and we should take the actual timestamp. Did you discuss with other devs about it?